tsz-compress/tests/compress-v2.rs

```diff
@@ -3,6 +3,8 @@ use tsz_compress::prelude::*;
 
 extern crate alloc;
 
+use rand::Rng;
```

Comment:
Prefer imports close to the module of use unless the intent is to export. This import would belong in the tests mod.

Reply:
Added changes in commit: 
Refactor: Enhance Code Idiomaticity and Readability ([42dcf40](https://github.com/qsib-cbie/tsz/pull/25/commits/42dcf407a455aab897eb5925d96ab7a3c0547021))